### PR TITLE
Remove Telldus Core service from openHABian

### LINKED ISSUE #1529: Remove Telldus
As the telldus package is no longer maintained, we should remove support for it from openHABian because it has become increasingly difficult to maintain. 

@mstormi WDYT

### PR DESCRIPTION
**Pull request by ecdye**
Fixes #1529

Signed-off-by: Ethan Dye <mrtops03@gmail.com>

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -2,6 +2,15 @@ Hit tab to unselect buttons and scroll through the text using UP/DOWN or
 PGUP/PGDN. All announcements are stored in `/opt/openhabian/docs/CHANGELOG.md`
 for you to lookup.
 
+## Telldus Core service removed ## May 20, 2021
+The Telldus Core service has now been removed from openHABian, and will no
+longer receive active support from the openHABian developers. Existing
+installations will be unaffected. The service was removed as it had become to
```
Comment (mstormi) [inline NEWS.md:8]:
```suggestion
installations will be unaffected. The service was removed as it had become too
```

File: functions/packages.bash [hunk 2]
```diff
@@ -24,7 +24,7 @@ samba_setup() {
   echo "OK"
 
   echo -n "$(timestamp) [openHABian] Setting up Samba service... "
-  if ! cond_redirect mkdir -p /var/log/samba /var/run/samba; then echo "FAILED (create directories)"; return 1; fi
```
Comment (mstormi) [inline functions/packages.bash:27]: please don't mix multiple issues in one PR

**Review by mstormi — APPROVED** [review]
essentially ok you can merge but mind my comment next time please

File: functions/packages.bash (hunk 2 above)
Comment (ecdye) [reply functions/packages.bash:27]: Whoops I for got that was in here, I'll remove it.

**Pull request merged**